java/org/contikios/cooja/plugins/ScriptRunner.java

```diff
@@ -413,16 +375,49 @@ private void updateTitle() {
     frame.setTitle(title);
   }
 
+  /** Check if the script has been updated and offer the user to save the changes. */
+  private void checkForUpdatesAndSave() {
+    if (!codeEditorChanged) {
+      return;
+    }
+    String s1 = "Yes";
+    String s2 = "No";
+    Object[] options = {s1, s2};
+    if (JOptionPane.showOptionDialog(Cooja.getTopParentContainer(),
+            "Do you want to save the changes to " + linkedFile.getName() + "?",
```

Comment:
What do you think about showing an absolute file path? I have many script files with similar names and would like to see exactly which file will be overwritten.

Reply:
Fixed.